sensor_msgs/PointCloud2[] cloud_list_in
---
sensor_msgs/PointCloud2 cloud_out
